cypress/e2e/collection.spec.js

```diff
@@ -0,0 +1,12 @@
+// Test for clicking collection item from home page 
+
+describe('Clicking into a collection ', () => {
+it('finds the content "type"', () => {
+    cy.visit("http://localhost:3000");
```

Comment:
We set our baseUrl as https://intern-x-marketplace.herokuapp.com/ in the `cypress.config.ts` file. 
```suggestion
    cy.visit("/");
```